book.rb

```diff
@@ -0,0 +1,19 @@
+# Represents a book object with a title, an author, and associated rentals
+class Book
+  attr_accessor :title, :author
+
```

Comment:
- [ ] Kindly inlude the Getters and Setters for `@rentals`.

## Sample code:
``attr_accessor :title, :author, :rentals``

Reply:
Hi @Shedrack-Sunday ✋,  Thanks a lot for your suggestion 👍 I will implement your suggestion before the next review. Thanks once again for reviewing my pull request.

Reply:
You're welcome